### PR TITLE
feat(route-monitor-operator): add release

### PR DESCRIPTION
**Pull request by georgettica**

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -293,6 +293,10 @@ approve:
   repos:
   - operator-framework/operator-lib
   require_self_approval: true
+- commandHelpLink: ""
+  repos:
+  - openshift/route-monitor-operator
+  require_self_approval: false
```
Comment (cblecker) [inline core-services/prow/02_config/_plugins.yaml:299]: Why is this here?

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -2593,6 +2597,9 @@ lgtm:
   - ansible-collections/community.okd
   - openshift-priv/community.okd
   review_acts_as_lgtm: true
+- repos:
+  - openshift/route-monitor-operator
+  review_acts_as_lgtm: true
```
Comment (cblecker) [inline core-services/prow/02_config/_plugins.yaml:2602]: Why is this here?

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
```
Comment (cblecker) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:5]: This should be UBI

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+canonical_go_repository: github.com/openshift/route-monitor-operator
+promotion:
+  name: "4.6"
+  namespace: ocp
```
Comment (cblecker) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:14]: Remove these

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
```
Comment (cblecker) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:24]: Remove these

Comment (RiRa12621) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:24]: Don't these mean that we follow a normal release cycle like other operators?

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml (hunk 4 above)
Comment (RiRa12621) [reply ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:14]: Don't these mean that we follow a normal release cycle like other operators?

File: core-services/prow/02_config/_plugins.yaml (hunk 2 above)
Comment (RiRa12621) [reply core-services/prow/02_config/_plugins.yaml:2602]: Is it a bad idea?
if someone clicks the lgtm button in the review, the label gets added, or not?

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (RiRa12621) [reply core-services/prow/02_config/_plugins.yaml:299]: I think @georgettica generated those with the makefile

**Comment by alvaroaleman** [comment]
/uncc
Because this spams my notifications. Feel free to re-assign me once this has a lgtm

File: ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- dockerfile_path: Dockerfile
+  from: os
+  inputs:
+    openshift_release_golang-1.14:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.14
+  to: route-monitor-operator
+promotion:
+  name: "4.7"
+  namespace: ocp
```
Comment (cblecker) [inline ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml:32]: We don't need/want a promotion stanza. This is only for cluster operators that are shipped as a part of OCP.

**Comment by georgettica** [comment]
I want to make the proccess better by working on this with you @bbguimaraes
